.circleci/config.yml

```diff
@@ -76,12 +76,18 @@ jobs:
           paths:
             - .
   checkout_tags:
-    <<: *build_config
+    <<: *deploy_config
```

Comment:
`deploy_config` and `build_config` are the same now. Can we remove one?

Reply:
I think it's worth keeping deploy config distinct from build config as it is just a coincidence that they are the same at the moment. No reason to assume that they would remain the same forever as they serve different purposes.